web/src/engine/osk/src/banner/suggestionBanner.ts

```diff
@@ -170,7 +176,22 @@ export class BannerSuggestion {
 
     if(suggestion && suggestion.displayAs) {
       const rawMetrics = getTextMetrics(suggestion.displayAs, format.emSize, format.styleForFont);
-      this._textWidth = rawMetrics.width;
+      let projectedHeight = 1;
+      if(rawMetrics.fontBoundingBoxAscent) {
+        projectedHeight = rawMetrics.fontBoundingBoxAscent + rawMetrics.fontBoundingBoxDescent;
+      }
```

Comment:
```suggestion
      let projectedHeight = (rawMetrics.fontBoundingBoxAscent) ? rawMetrics.fontBoundingBoxAscent + rawMetrics.fontBoundingBoxDescent : 1;
```

Reply:
And if we do that, then `const`. Let's start using const more in our ts code (we can turn on a lint rule for that; we already have in kmc)

```suggestion
      const projectedHeight = rawMetrics.fontBoundingBoxAscent 
        ? rawMetrics.fontBoundingBoxAscent + rawMetrics.fontBoundingBoxDescent
        : 1;
```